### ISSUE TITLE
LvimInfo fails to run

### ISSUE THREAD

**Issue by danielo515** [LunarVim/LunarVim#2900]
### Problem description

I just updated to the latest rolling, and when I do LvimInfo it reports an error.

### LunarVim version

rolling-9caa457

### Neovim version (>= 0.7.2)

NVIM v0.7.0-dev+882-g9304ee387

### Operating system/version

macOS 12.4

### support info

```
14:47:02 [WARN ] lvim: "Invalid configuration: ...ielo/.local/share/lunarvim/lvim/lua/lvim/lsp/manager.lua:91: module 'mason-lspconfig.mappings.server' not found:\n\tno field package.preload['mason-lspconfig.mappings.server']\n\tno file './mason-lspconfig/mappings/server.lua'\n\tno file '/usr/local/Cellar/luajit-openresty/2.1-20220411/share/luajit-2.1.0-beta3/mason-lspconfig/mappings/server.lua'\n\tno file '/usr/local/share/lua/5.1/mason-lspconfig/mappings/server.lua'\n\tno file '/usr/local/share/lua/5.1/mason-lspconfig/mappings/server/init.lua'\n\tno file '/usr/local/Cellar/luajit-openresty/2.1-20220411/share/lua/5.1/mason-lspconfig/mappings/server.lua'\n\tno file '/usr/local/Cellar/luajit-openresty/2.1-20220411/share/lua/5.1/mason-lspconfig/mappings/server/init.lua'\n\tno file './mason-lspconfig/mappings/server.so'\n\tno file '/usr/local/lib/lua/5.1/mason-lspconfig/mappings/server.so'\n\tno file '/usr/local/Cellar/luajit-openresty/2.1-20220411/lib/lua/5.1/mason-lspconfig/mappings/server.so'\n\tno file '/usr/local/lib/lua/5.1/loadall.so'\n\tno file './mason-lspconfig.so'\n\tno file '/usr/local/lib/lua/5.1/mason-lspconfig.so'\n\tno file '/usr/local/Cellar/luajit-openresty/2.1-20220411/lib/lua/5.1/mason-lspconfig.so'\n\tno file '/usr/local/lib/lua/5.1/loadall.so'" file="init.lua", line=119
"chezmoi/dot_config/lvim/config.lua" 61L, 1992B
"chezmoi/dot_config/lvim/lua/user/keymaps.lua" 70L, 3450B
E5108: Error executing lua vim/shared.lua:0: t: expected table, got string
stack traceback:
	[C]: in function 'error'
	vim/shared.lua: in function 'validate'
	vim/shared.lua: in function <vim/shared.lua:0>
	vim/shared.lua: in function 'tbl_filter'
	...anielo/.local/share/lunarvim/lvim/lua/lvim/core/info.lua:115: in function 'make_auto_lsp_info'
	...anielo/.local/share/lunarvim/lvim/lua/lvim/core/info.lua:159: in function 'toggle_popup'
	[string ":lua"]:1: in main chunk
E5108: Error executing lua vim/shared.lua:0: t: expected table, got string
stack traceback:
	[C]: in function 'error'
	vim/shared.lua: in function 'validate'
	vim/shared.lua: in function <vim/shared.lua:0>
	vim/shared.lua: in function 'tbl_filter'
	...anielo/.local/share/lunarvim/lvim/lua/lvim/core/info.lua:115: in function 'make_auto_lsp_info'
	...anielo/.local/share/lunarvim/lvim/lua/lvim/core/info.lua:159: in function 'toggle_popup'
	[string ":lua"]:1: in main chunk
```

**Comment by kylo252** [comment]
I can't really reproduce this, but you seem to be missing `mason-lspconfig`, so I'm guessing that `:PackerSync` didn't run correctly.

**Comment by danielo515** [comment]
@kylo252 thanks for your answer. I run packersync several times, and sometimes it stucks. 
I narrowed down my problem (I think) to my lsp config file, which may not be correct for use it with the new mason dependency/migration?

```lua
local M = {}

local status_ok, nls = pcall(require, "null-ls")
if not status_ok then
  return
end

function M.config()
  local formatters = require "lvim.lsp.null-ls.formatters"
  formatters.setup {
    { name = "eslint_d" },
    { name = "stylua" },
  }

  local linters = require "lvim.lsp.null-ls.linters"
  linters.setup {
    { command = "eslint", filetypes = { "typescript", "typescriptreact" } },
  }
  local graphql_lsp_opts = {
    filetypes = { "graphql", "typescriptreact", "javascriptreact", "typescript" },
  }

  require("lvim.lsp.manager").setup("graphql", graphql_lsp_opts)
  -- require("lvim.lsp.manager").setup "marksman" -- this fails with mason
  require("lvim.lsp.manager").setup("jsonls", {
    settings = {
      json = {
        schemas = require("schemastore").json.schemas(),
        validate = { enable = true },
      },
    },
  })
  local setup_code_actions = require("lvim.lsp.null-ls.code_actions").setup
  local refactorin_opts = nls.builtins.code_actions.refactoring.with {
    filetypes = { "typescript", "javascript", "lua", "c", "cpp", "go", "python", "java", "php" },
  }
  setup_code_actions { refactorin_opts }
end

return M

```

**Comment by kylo252** [comment]
@danielo515, why the duplicate setup for [`jsonls`](https://github.com/LunarVim/LunarVim/blob/9caa457810a26369f848d4f90f6e0ec84e0048e0/lua/lvim/lsp/providers/jsonls.lua)?

otherwise, I don't see anything else wrong going on. was it resolved with https://github.com/danielo515/dotfiles/commit/fca71448a7d9bd41618c1f647c9add0ff54d6bca?

**Comment by danielo515** [comment]
> @danielo515, why the duplicate setup for [`jsonls`](https://github.com/LunarVim/LunarVim/blob/9caa457810a26369f848d4f90f6e0ec84e0048e0/lua/lvim/lsp/providers/jsonls.lua)?

To enable validation. Also, since it is something I use I prefer to specify it clearly.

> I don't see anything else wrong going on. was it resolved with [danielo515/dotfiles@fca7144](https://github.com/danielo515/dotfiles/commit/fca71448a7d9bd41618c1f647c9add0ff54d6bca)?

They only way I resolved it was by not requiring my file, but that was before I solved some problems with packer, so I will try again

**Comment by danielo515** [comment]
The command does not fail anymore, but now I don't get typescript files attach to any server. I'll try to comment the require again and test
